pkg/agent/ip.go

```diff
@@ -113,7 +113,7 @@ func findAddress(addrs []net.Addr, ipType string) (net.IP, bool) {
 }
 
 func getIP(pip net.IP, ipType string) (net.IP, bool) {
-	if pip == nil || pip.IsLoopback() || pip.IsLinkLocalUnicast() {
+	if pip == nil || pip.IsLoopback() {
```

Comment:
why did u remove LL check ? this will break ipv6 cluster only on baremetal ?

Reply:
This was not an intentional change. Reverting.